plugins/modules/gitlab_group.py

```diff
@@ -279,10 +286,13 @@ def update_group(self, group, arguments):
 
         return (changed, group)
 
-    def delete_group(self):
+    '''
+    @param force To delete even if projects inside
+    '''
+    def delete_group(self, force=False):
         group = self.group_object
 
-        if len(group.projects.list(all=False)) >= 1:
+        if not force and len(group.projects.list(all=False)) >= 1:
             self._module.fail_json(
                 msg="There are still projects in this group. These needs to be moved or deleted before this group can be removed.")
```

Comment:
How about extending this message to mention that `force_delete=true` can still delete them?

Reply:
Good idea, I add it.